python/cuml/dask/datasets/blobs.py

```diff
@@ -47,7 +47,7 @@ def make_blobs(n_samples=100, n_features=2, centers=None, cluster_std=1.0,
                n_parts=None, center_box=(-10, 10), shuffle=True,
                random_state=None, return_centers=False,
                verbose=False, order='F', dtype='float32',
-               client=None):
+               workers=None, client=None):
```

Comment:
I would suggest adding the new argument after client just to alleviate the potential of breaking any users who are currently passing in a `client` without explicitly naming it.